extensions/replication/tasks/ReplicateObject.js

```diff
@@ -373,6 +374,13 @@ class ReplicateObject extends BackbeatTask {
                 return doneOnce(err);
             }
             partObj.setDataLocation(data.Location[0]);
+            const extMetrics = {};
+            extMetrics[this.site] = {
+                ops: 1,
+                bytes: partObj.getPartSize(),
+            };
+            this.mProducer.publishMetrics(
+                extMetrics, metricsTypeProcessed, metricsExtension, () => {});
```

Comment:
Maybe needed also in `MultipleBackendTask`

Reply:
Correct me if I'm wrong, normally MultipleBackendTask is dead code for 7.4, this class was included when we started developing zenko on top of S3C code but should normally not be used by running processes. Since this PR is a no-op for 8.0 and 8.1 (as the multiple backend metrics support is already there and I preferred a minimal functional PR than a complete backport), I think we can leave as is.

Reply:
I believe the MultipleBackendTask was added before zenko, or at least, it wasn't added specifically for zenko. If I recall it was the release in which we introduced CRR to public clouds, starting with AWS.